src/Sylius/Behat/.gitignore

```diff
@@ -0,0 +1 @@
+/phpspec.yml
```

Comment:
This way I can call `../../../bin/phpspec run` from that directory and have only that specifications runned, what results in a faster feedback cycle.

IMO ignoring absolute paths is better than any file named `phpspec.yml`, I only want to ignore `src/Sylius/Behat/phpspec.yml` but not `src/Sylius/Behat/foo/bar/phpspec.yml`.